### DISCUSSION TITLE
Error updating to 2.0.0b64: manifest unknown

### DISCUSSION THREAD

**Discussion by jhansche** [music-assistant/support#1434] [Q&A] [answered]
Trying to update from 2.0.0b63 fails, reporting:
```
[139792573204032] Error updating Music Assistant BETA: Can't install ghcr.io/music-assistant/server:2.0.0b64: 500 Server Error for http+docker://localhost/v1.42/images/create?tag=2.0.0b64&fromImage=ghcr.io%2Fmusic-assistant%2Fserver&platform=linux%2Famd64: Internal Server Error ("manifest unknown")
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/hassio/update.py", line 167, in async_install
    await async_update_addon(self.hass, slug=self._addon_slug, backup=backup)
  File "/usr/src/homeassistant/homeassistant/components/hassio/handler.py", line 52, in _wrapper
    raise HassioAPIError(data["message"])
homeassistant.components.hassio.handler.HassioAPIError: Can't install ghcr.io/music-assistant/server:2.0.0b64: 500 Server Error for http+docker://localhost/v1.42/images/create?tag=2.0.0b64&fromImage=ghcr.io%2Fmusic-assistant%2Fserver&platform=linux%2Famd64: Internal Server Error ("manifest unknown")
```

Any idea what this means or how to fix it?

**Comment by jhansche** [comment]
Looks like there is no 2.0.0b64 available?
https://github.com/music-assistant/server/pkgs/container/server

Is this a mistake, or some kind of cache that takes a while to propagate?

Is there any way to prevent Home Assistant from offering an update if it will obviously fail?

**Reply by OzGav** [reply, marked as answer]
There was a github failure. It is being worked on (not HAs fault it is GitHub)

**Reply by jhansche** [reply]
Sure, that makes sense.

I guess what I mean more so is: is it possible to wait until the image is available on github before the new version is tagged and made available for HA to update at all? Or is the tag what kicks off both, building/deploying the docker image _and_ signalling to HA that an update is available? Just seems a little catch-22-y 🤔

**Reply by OzGav** [reply]
As I understand it for some reason the build failed but GitHub created the tags.

**Reply by OzGav** [reply]
beta65 released

**Reply by marcelveldt** [reply]
@jhansche yeah sure possible so if you are bored, feel free to help us out with an automation to do that ;-) 
Until then I just take my chances as this happens only 1% of the time and we're still beta.
